bunq/sdk/json/installation_context_adapter.py

```diff
@@ -0,0 +1,67 @@
+from bunq.sdk.context import api_context
+from bunq.sdk.json import converter
+from bunq.sdk.security import security
+
+
+class InstallationContextAdapter(converter.JsonAdapter):
+    # Attribute/Field constants
+    _ATTRIBUTE_TOKEN = '_token'
+    _FIELD_TOKEN = 'token'
+
+    _ATTRIBUTE_PRIVATE_KEY_CLIENT = '_private_key_client'
+    _FIELD_PRIVATE_KEY_CLIENT = 'private_key_client'
+
+    _ATTRIBUTE_PUBLIC_KEY_CLIENT = '_public_key_client'
+    _FIELD_PUBLIC_KEY_CLIENT = 'public_key_client'
+
+    _ATTRIBUTE_PUBLIC_KEY_SERVER = '_public_key_server'
+    _FIELD_PUBLIC_KEY_SERVER = 'public_key_server'
+
+    @classmethod
+    def deserialize(cls, target_class, obj):
+        """
+        :type target_class: api_context.InstallationContext|type
```

Comment:
same story as before, this is the class right, not an instance?
with "modern" type annotations that would then be `Type[Foo]`

Reply:
Can you please give an example here? Do you mean I should replace the type (`api_context.InstallationContext|type`) with `Type[InstallationContext]`?